src/tests/sqlsmith/src/lib.rs

```diff
@@ -87,6 +87,9 @@ struct SqlGenerator<'a, R: Rng> {
     /// We might not read from all tables.
     bound_relations: Vec<Table>,
 
+    /// Relations Outer corresponding to current subquery
```

Comment:
Don't quite understand what is `Relations Outer`. Could you leave an example?

Reply:
ignore it, because I change my plan just now. I will edit the docs later.